typescript/t.d.ts

```diff
@@ -46,8 +46,8 @@ type WithOrWithoutPlural<Key> = _JSONFormat extends 'v4' | 'v3'
   ? Key extends `${infer KeyWithoutOrdinalPlural}${_PluralSeparator}ordinal${_PluralSeparator}${PluralSuffix}`
     ? KeyWithoutOrdinalPlural | Key
     : Key extends `${infer KeyWithoutPlural}${_PluralSeparator}${PluralSuffix}`
-    ? KeyWithoutPlural | Key
-    : Key
+      ? KeyWithoutPlural | Key
```

Comment:
Tiny nit - did these come built this way? If not, I think we may want to just revert the `t.d.ts`and `t.v4.d.ts` for a simpler diff since the only changes are indentation changes